packages/native-stack/src/types.tsx

```diff
@@ -351,6 +375,21 @@ export type NativeStackNavigationOptions = {
    * @platform ios
    */
   statusBarStyle?: ScreenProps['statusBarStyle'];
+  /**
+   * Sets the translucency of the status bar. Defaults to `false`.
+   *
+   * @platform android
+   */
+  statusBarTranslucent?: boolean;
```

Comment:
This prop doesn't seem to work at all in my emulator neither in expo managed nor bare workflow. Maybe something with my device but could you double-check this? 🙏 

I'll test try to test this more

Reply:
It was working for when I tested it, but only when you're not using StatusBar from react-native or expo IIRC